### PR TITLE
[INSTALL] list required root packages in debian

### PR DESCRIPTION
**Pull request by fredRos**
@oschulz feel free to add the required packages for building on ubuntu as per #109

### CODE REVIEW

**Comment by oschulz** [comment]
I'll hold off on this until we've decided on ROOT version compatibility.

**Comment by oschulz** [comment]
This isn't really critical for version 1.0, so I'm removing the milestone.

**Comment by fredRos** [comment]
@oschulz when you test this, please read through the updated `INSTALL.md` and check if it is enough to build on a clean ubuntu. Regarding ROOT, I"m sure it's right but I don't what other build tools might be required since I have all that already installed on my testing system.

File: INSTALL.md [hunk 1]
```diff
@@ -7,37 +7,43 @@ BAT on your computer.
...
-BAT has been developed on Linux machines running different
-distributions and different versions of the kernel and gcc. As far as
-we know there is nothing distribution-dependent inside of BAT. A gcc
-version >4.2 should suffice to compile the C++ code.
-
-The installation and functionality of BAT has also been tested on Mac
-OS X with gcc and clang.
+BAT has been developed on Linux. The installation, unit tests, and
+examples are run and known to work on Linux and Mac OS X with both gcc
+and clang.
```
Comment (oschulz) [inline INSTALL.md]: To be fair, we run the tests with GCC on Linux and Clang on OS-X, not with both compilers on both platforms.

**Comment by oschulz** [comment]
Ok, I just tested this on a Docker instance with Ubuntu 14:04. The package list is fine, everything works. However, if people want to download BAT via git-clone instead of getting a tarball (increasingly popular nowadays), they'll also need the following packages

    git-core autoconf automake libtool

Maybe we should add that do the install doc. git-core most people will already have, but sadly, autoconf, automake and libtool are not automatically installed when installing build-essential.

**Comment by fredRos** [comment]
> To be fair, we run the tests with GCC on Linux and Clang on OS-X, not with both compilers on both platforms.

I used clang on linux, too. It shows some more warnings in the debug mode so it's useful but it doesn't support openMP unfortunately. I updated the instructions as you requested

> Maybe we should add that do the install doc.

I just did, please proceed and merge

**Comment by oschulz** [comment]
Fails to build (all platforms / ROOT versions):

```
../../../models/base/BCEfficiencyFitter.cxx: In constructor 'BCEfficiencyFitter::BCEfficiencyFitter(const TH1&, const TH1&, const TF1&, const string&)':
../../../models/base/BCEfficiencyFitter.cxx:55:13: error: no match for 'operator=' (operand types are 'TH1D' and 'const TH1')
     fTrials = trials;
```

**Comment by fredRos** [comment]
Oh sorry about that, I had reverted to something impossible but checked only with 5.34.14.

Now I fixed it and tested it myself with 5.34.14 and 5.34.19 and it works with both versions. Please merge as quickly as possible

**Pull request merged**